nautobot/extras/api/views.py

```diff
@@ -343,7 +344,7 @@ def _create_schedule(serializer, data, commit, job, job_model, request):
     elif type_ == JobExecutionType.TYPE_CUSTOM:
         time = serializer.get("start_time")  # doing .get("key", "default") returns None instead of "default"
         if time is None:
-            time = timezone.now()
+            time = timezone.now() + timedelta(seconds=20)
```

Comment:
I seem to recall you mentioning previously why you added 20 seconds, but would you please mind adding a brief comment here for posterity?